rq/job.py

```diff
@@ -860,6 +872,8 @@ def restore(self, raw_data) -> Any:
             except Exception:
                 self._result = "Unserializable return value"
         self.timeout = parse_timeout(obj.get('timeout')) if obj.get('timeout') else None
+        self.success_callback_timeout = parse_timeout(obj.get('success_callback_timeout')) if obj.get('timeout') else CALLBACK_TIMEOUT
```

Comment:
This is minor, but it would be better if we nest related parameters together so it's easier to reason about. At the moment it looks like this:
```
{
    "success_callback_name": "do_something",
    "success_callback_timeout": 60
}
```
If we nest variables:
```
{
    "success_callback": {
        "name": "do_something",
        "timeout": 60
    }
}
```